tests/topotests/bgp_set_aspath_exclude/test_bgp_set_aspath_exclude.py

```diff
@@ -134,12 +135,66 @@ def _bgp_regexp_1(router):
         }
     }
 
+    # tgen.mininet_cli()
     test_func = functools.partial(_bgp_regexp_1, tgen.gears["r1"])
     _, result = topotest.run_and_expect(test_func, None, count=30, wait=0.5)
 
     assert result is None, "Failed overriding incoming AS-PATH with regex 2 route-map"
 
 
+def test_no_bgp_set_aspath_exclude_access_list():
+    tgen = get_topogen()
+
+    if tgen.routers_have_failure():
+        pytest.skip(tgen.errors)
+
+    rname = "r1"
+    r1 = tgen.gears[rname]
+
+    r1.vtysh_cmd(
+        """
+conf
+ no bgp as-path access-list SECOND permit 2
+    """
+    )
+
+    expected = {
```

Comment:
Please move `expected` stuff directly into the function.

Reply:
taken into account